packages/@aws-cdk/tmp-toolkit-helpers/src/api/io/payloads/bootstrap-environment-progress.ts

```diff
@@ -1,6 +1,12 @@
 import type * as cxapi from '@aws-cdk/cx-api';
 
 export interface BootstrapEnvironmentProgress {
+  /**
+   * Uniquely identifies this marker amongst concurrent messages
+   *
+   * This is an otherwise meaningless identifier.
+   */
+  readonly marker: string;
```

Comment:
It would be nicer if `marker` was part of the message structure itself (like `data` is or isn't), rather than part of the payload.